test/fixtures/core.scale/label-offset-vertical-axes.json

```diff
@@ -0,0 +1,40 @@
+{
+    "tolerance": "0.008",
```

Comment:
`0.008` might be too high, did you verify locally that the test fails if you temporary change `"labelOffset": 25` to `20`? Maybe it worths a try to `0.005`, the lower the better :)

@etimberg I'm curious to know if there is a font that can be consistently rendered on all platforms/browsers that we could use for our unit tests.